components/nexus-security/src/main/java/org/apache/shiro/nexus5727/FixedDefaultWebSessionManager.java

```diff
@@ -36,15 +36,20 @@
 public class FixedDefaultWebSessionManager
     extends DefaultWebSessionManager
 {
+  /**
+   * Default HTTP session cookie name that Nexus uses - NXSESSIONID
+   */
+  public static final String DEFAULT_NEXUS_SESSION_COOKIE_NAME = "NXSESSIONID";
+
   private static final Logger log = LoggerFactory.getLogger(FixedDefaultWebSessionManager.class);
 
   @Inject
   public void configureProperties(
       final @Named("${shiro.globalSessionTimeout:-" + DEFAULT_GLOBAL_SESSION_TIMEOUT + "}") long globalSessionTimeout,
-      final @Named("${shiro.secureSessionCookies:-false}") boolean secureSessionCookies)
```

Comment:
@mcculls mentioned he though to leave this in place just in case.

I think there is no use case for leaving it personally. isSecure() should do the right thing and we already know Secure flag on access over non-secure connection will break the UI.